b2g.sh

```diff
@@ -59,13 +59,18 @@ if [ -f "$B2G_DIR/libdmd.so" ]; then
   LD_PRELOAD="$B2G_DIR/libdmd.so $LD_PRELOAD"
   export DMD="1"
 fi
-if [ -f "$B2G_DIR/libmozsandbox.so" ]; then
-    echo "Running with Sandbox."
-    LD_PRELOAD="$B2G_DIR/libmozsandbox.so $LD_PRELOAD"
-fi
+#if [ -f "$B2G_DIR/libmozsandbox.so" ]; then
+#    echo "Running with Sandbox."
+#    LD_PRELOAD="$B2G_DIR/libmozsandbox.so $LD_PRELOAD"
+#fi
```

Comment:
Let's merge it and address this later.

Reply:
The purpose of preloading libmozsandbox.so is fixing the problem of symbols resolving. On some devices, drivers or system libraries may change the order of shared objects and cause incorrect out comes of sandbox hooks.  Without preloading, it is still good for most of case, but some platforms may break sandbox.